.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseDateExtractorConfiguration.cs

```diff
@@ -75,38 +75,25 @@ public class ChineseDateExtractorConfiguration : AbstractYearExtractor, IDateTim
             // (2015年)?(农历)?十月二十(星期三)?
             new Regex(DateTimeDefinitions.DateRegexList3, RegexFlags),
 
-            DateTimeDefinitions.DefaultLanguageFallback == Constants.DefaultLanguageFallback_DMY ?
+            // 7/23
+            new Regex(DateTimeDefinitions.DateRegexList4, RegexFlags),

```

Comment:
Why is this no longer under the flag check?

Reply:
For now, DataRegexList4 only extract "X/X" pattern,  in Chinese, only use M/D in "X/X" pattern according to the rules.